### PR TITLE
Infinity gauntlet or whatever for Jazz's bus maybe I'll also put the gems in fishing.

### PR DESCRIPTION
**Pull request by cowbot92**
Full credit for all sprites and original idea go to the original PR https://github.com/yogstation13/Yogstation/pull/6619

# Document the changes in your pull request

This PR adds the infinity gauntlet and 6 stones, with corresponding spells.

Mind
Soul
Power
Space
Time
Reality

once collecting all the stones you'll gain access to the "snap" spell, killing half of all existing life in the universe.

# Why is this good for the game?

something for an event for jazz

# Testing

will post a vid once done

# Spriting

refer to https://github.com/yogstation13/Yogstation/pull/6619

# Changelog

:cl:   Molti & Cowbot92 (Also thanks to Chubby for helping me fix time stop after he broke it) (Actually it's pretty much entirely Molti)
rscadd: Adds the dated marvel reference

/:cl:

### CODE REVIEW

**Comment by neeshacark** [comment]
https://youtu.be/jF0E2wnZ-A8 all over again

**Comment by Addust** [comment]
remember to undefine lowmemorymode once this is out of draft

**Comment by ToasterBiome** [comment]
reopen when you work on this

**Comment by cowbot92** [comment]
this is done

**Comment by GDRdev** [comment]
*stare

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```
Comment (Moltijoe) [inline _maps/_basemap.dm]: don't forget this

**Review by Moltijoe — DISMISSED** [review]

File: _maps/_basemap.dm (hunk 1 above)
Comment (cowbot92) [reply _maps/_basemap.dm]: Haha im dumb

**Review by cuackles — APPROVED** [review]

**Pull request closed without merging**